server/routes/admin/licence.ts

```diff
@@ -112,7 +112,7 @@ export = (licenceService, signInService, prisonerService, audit, roNotificationH
     asyncMiddleware(async (req, res) => {
       const { abookingId: bookingId } = req.params
       const systemToken = await signInService.getClientCredentialsTokens(req.user.username)
-      const [ignored, error] = unwrapResult(
+      const [, error] = unwrapResult(
```

Comment:
new eslint rule not liking unused variable

Reply:
then still needs comma?